### ISSUE TITLE
Settings Button, A user must have a default settings object

### ISSUE THREAD

**Issue by adle29** [django-wiki/django-wiki#577]
When I press the settings of an article I get the following error. 

```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8000/_settings/

Django Version: 1.10.3
Python Version: 3.5.2
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.sites',
 'django.contrib.humanize',
 'django_nyt',
 'mptt',
 'sekizai',
 'sorl.thumbnail',
 'wiki',
 'wiki.plugins.attachments',
 'wiki.plugins.notifications',
 'wiki.plugins.images',
 'wiki.plugins.macros']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/db/models/query.py" in get_or_create
  473.             return self.get(**lookup), False

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/db/models/query.py" in get
  385.                 self.model._meta.object_name

During handling of the above exception (Settings matching query does not exist.), another exception occurred:

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  39.             response = get_response(request)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  23.                 return view_func(request, *args, **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/wiki/decorators.py" in wrapper
  177.         return func(request, article, *args, **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/wiki/views/article.py" in dispatch
  632.             **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/wiki/views/mixins.py" in dispatch
  35.         return super(ArticleMixin, self).dispatch(request, *args, **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/views/generic/base.py" in dispatch
  88.         return handler(request, *args, **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/wiki/views/article.py" in get
  682.             self.forms.append(Form(new_article, self.request))

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/wiki/plugins/notifications/forms.py" in __init__
  146.             interval=notify_settings.INTERVALS_DEFAULT

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/db/models/manager.py" in manager_method
  85.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/db/models/query.py" in get_or_create
  475.             return self._create_object_from_params(lookup, params)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/db/models/query.py" in _create_object_from_params
  505.                 obj = self.create(**params)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django/db/models/query.py" in create
  399.         obj.save(force_insert=True, using=self.db)

File "/Users/ccs_admin/Documents/Abraham/copies/test/venv/lib/python3.5/site-packages/django_nyt/models.py" in save
  122.                 raise ValueError("A user must have a default settings object")

Exception Type: ValueError at /_settings/
Exception Value: A user must have a default settings object
```

**Comment by benjaoming** [comment]
Thanks for reporting this!

You are right, this is an actual issue and was fixed in 9ca768f05d4a55238b1b84dc3e1861e72c4d9e2c

Should be serious enough to release an update asap... let me get to that today!

**Comment by benjaoming** [comment]
This is fixed in 0.2b2, but please do help out by confirming/disconfirming :)

Thanks!

**Comment by adle29** [comment]
Yes, fixed! Thanks a lot Benjamin.  Issue closed.
